apps/omg_db/mix.exs

```diff
@@ -50,4 +50,19 @@ defmodule OMG.DB.MixProject do
       _ -> []
     end
   end
+
+  defp version_and_git_revision_hash do
```

Comment:
since this code grew a bit, maybe it's worth to DRY it out?

But maybe more importantly: I'm not super-convinced if this isn't going to be difficult to work with in practice - VERSION file is going to be always present in the git changesets, it will pop up as conflicting very often, and it still will be rather easy to push out a wrong one eventually.

Isn't there an alternative to:
  - leave `VERSION` as it is (just `0.2.2` etc.)
  - append the git sha later - e.g. fetch it during compilation and store it in the `config` and serve from there?

Reply:
I don't think there's any place where we could put it that would work with how releases fetch versions when they build applications. We had a similar solution before and it didn't work out. 

For the second part:
I was thinking the same and started out with a git pre-commit hook that always puts VERSION back from master. But you can still change it if your run it with `--no-verify`

Let me see what else I can do...